bot/__init__.py

```diff
@@ -241,7 +243,7 @@ def parse_python(buffer_pos):
     self.index += pos
 
     # If the command looks like a python syntax command, try to parse it.
-    if current == "(" or current == "[":
+    if current and current == "(" or current == "[":
```

Comment:
Is this needed? if `current` is a string, `bool(current) = len(current) > 0`. If `current` is `None`, as above, then `current != "("` and `current != "["`, or am I missing something?

Reply:
yep, you're right.